### PR TITLE
Update version to 0.11.1

### PR DESCRIPTION
**Pull request by aleokdev**
Preparing for new crates.io release

### CODE REVIEW

**Review by bjorn — DISMISSED** [review]

**Comment by bjorn** [comment]
Regarding changelog update, I think it might be worth still adding the removal of the function #266 is about to 0.11? I guess it's covered by "VFS support", but I think it could use explicit mentioning under the Removed section.

**Comment by raimundomartins** [comment]
Just FYI, you name the struct `EmbeddedResourceReader` first and then `BytesResourceReader`. Would be better to just pick one.

**Comment by aleokdev** [comment]
Agh crap, thought I changed that, thanks

File: CHANGELOG.md [hunk 1]
```diff
@@ -30,6 +30,38 @@ objects. As such, `ResourceCache` has now methods for both getting and inserting
...
+impl<R: Read> tiled::ResourceReader for EmbeddedResourceReader<R> {
+    type Resource = R;
+    type Error = std::io::Error;
+
+    fn read_from(&mut self, path: &Path) -> std::result::Result<Self::Resource, Self::Error> {
+        if path == "the_path_to_load.tmx" {
+            if let Some(x) = self.reader.take() {
+                Ok(x)
+            } else {
+            Err(std::io::Error::from(ErrorKind::ResourceBusy))
```
Comment (bjorn) [inline CHANGELOG.md]: Indentation seems off?

**Review by bjorn — DISMISSED** [review]

**Review by bjorn — APPROVED** [review]

**Pull request merged**